test/core/end2end/h2_ssl_cert_test.cc

```diff
@@ -369,12 +381,38 @@ int main(int argc, char** argv) {
 
   grpc_init();
   ::testing::InitGoogleTest(&argc, argv);
+  while (--argc > 0) {
+      if (**++argv == '-') {
```

Comment:
Yes, you have to pass "engine" and then the engine name. 

`h2_ssl_cert_test -engine e_passthrough`

The reason I kept it seperate instead of doing it automatically as I am unsure of how gRPC tests work. As engine is not part of boringSSL, it will fail. So wanted to test this only in condition when gRPC is built with openSSL. Plus not sure what different kind of platforms gRPC test framework runs on. Hence the conservative route.

Reply:
@nicolasnoble Could you advice how to test ENGINE support in gRPC, where ENGINE only works for OpenSSL, but not BoringSSL.